src/stats/distributions.jl

```diff
@@ -26,22 +26,61 @@ end
 # -----------------------------------------------------------------------------
 # qqplots (M. K. Borregaard implementation from StatPlots)
 
-@recipe(QQNorm) do scene
-    default_theme(scene, Scatter)
-end
+"""
+    qqplot(x, y; kwargs...)
+Draw a Q-Q plot, comparing quantiles of two distributions. `y` must be a list of
+samples, i.e., `AbstractVector{<:Real}`, whereas `x` can be
+- a list of samples,
+- an abstract distribution, e.g. `Normal(0, 1)`,
+- a distribution type, e.g. `Normal`.
+In the last case, the Q-Q plot by fitting that distribution type to the data `y`.
+
+The attribute `qqline` (defaults to `:R`) determines how to compute a fit line for the Q-Q plot.
+Possible values are the following.
+- `:identity` draws the identity line (useful to see `x` and `y` follow the same distribution).
+- `:fit` fits the line to the quantile pairs (useful to see if the distribution of `y` can be obtained from the distribution of `x` via an affine transformation).
```

Comment:
Yes, I'm renaming `:quantile` to `:robust` (that's how Base R calls it, see [here](https://www.rdocumentation.org/packages/EnvStats/versions/2.3.1/topics/qqPlot)) under `qq.line.type`). They say `"least squares"` instead of `:fit`, but I think that's a bit too technical.